packages/runtime/src/widgets/Dropdown.tsx

```diff
@@ -18,16 +20,25 @@ interface SelectOption {
 
 const Dropdown: React.FC<DropdownProps> = (props) => {
   const [selectedValue, setSelectedValue] = useState<string>();
+  const handleChange = (value: string): void => {
```

Comment:
not necessary to declare a new function here you can just pass the setSelectedValue directly to the prop